src/cuga/backend/loops/service.py

```
@@ -0,0 +1,240 @@
+
+    def _arm_loop(self, loop: Loop):
+        """Register an APScheduler job for a loop. Returns the Job (or None)."""
+        from apscheduler.triggers.cron import CronTrigger
+        from apscheduler.triggers.interval import IntervalTrigger
+        from apscheduler.triggers.date import DateTrigger
+        from datetime import datetime, timedelta
+
+        if loop.status != LoopStatus.ACTIVE:
+            return None
+        if loop.expires_at and loop.expires_at < time.time():
+            asyncio.create_task(self.registry.update_status(loop.id, LoopStatus.EXPIRED))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)